Akkadu/rsi-widgets
# feature: interpretation manager

Please describe the purpose of the pull request.

* [notion page](https://www.notion.so/akkadu/RSI-SDK-interpretation-Manager-9c28a599dde54c79a35d39a7be4c55ba)
* I made the interpretation manager for react/vueJs/ vanillaJS
* This PR doesnt include the API lambda url to create an event.


## Motivation and Context



## How To Test

* Go on the vue-demo, you should see the interpretation-manager : https://deploy-preview-15--rsi-akkadu-vue-demo.netlify.app/
* you shouldn't be able to create an event IF:
  * You didnt' choose a language for this event
  * you didn't choose at least 2 interpretation languages.
  * when switching to `I have my own interpreters`
    * you should at least add 2 emails by interpretation languages, otherwise you will get a feedback.

  

## Screenshots (if appropriate)

## Checklist



加油！
